### ISSUE TITLE
no zwave updates is sent to mqtt

### ISSUE THREAD

**Issue by reiniertc** [OpenZWave/Zwave2Mqtt#880]
in Home Assistant, mosquito is setup and works (I see messages using MQTT explorer)
I have setup the zwave part and the mqtt part in zwave2mqtt.
my zwave-devices are being populated in zwave2mqtt
in MQTT explorer, i See my zwave and _EVENTS.
in _EVENTS I only see ZWAVE_GATEWAY-MQTT_gateway

But after that, no zwave changes (switch state, sensor values) are being send to MQTT (under _EVENTS of anywhere else).

What could be wrong?

**Comment by robertsLando** [comment]
@reiniertc Could you try to use: https://github.com/zwave-js/zwavejs2mqtt?

**Comment by reiniertc** [comment]
@robertsLando can I add this via Supervisor?

**Comment by robertsLando** [comment]
What is that?

**Comment by reiniertc** [comment]
in home assistant, there is an option "supervisor" which let's you install add-ons via a user-interface

**Comment by reiniertc** [comment]
let me rephrase my question: is zwave-js2mqtt a home assistant addon?

**Comment by robertsLando** [comment]
Not yet, but I have a tutorial: https://zwave-js.github.io/zwavejs2mqtt/#/guide/homeassistant?id=accessing-zwavejs2mqtt-from-within-home-assistant

**Comment by reiniertc** [comment]
mmm, ok thanks. 
I'm not experienced enough to install a custom add on like this one on my raspberry pi4.
That's why I have chosen to only install addons via the add-on functions of homeassistant via Supervisor.

You advised me to install zwavejs2mqtt for a reason: why is that better than zwave2mqtt I wonder?

**Comment by robertsLando** [comment]
You will find main reasons here: https://zwave-js.github.io/zwavejs2mqtt/#/getting-started/why

Additionally OZW main author isn't very active in last 4/5 months and said that he will not be able to be so active in the future. I cannot say OZW is died but most platforms are moving to zwavejs now
